Update custom rest endpoint documentation and example (#26149)
keycloak/keycloak: examples/providers/rest/README.md

Example Realm REST Resource provider
====================================

You can deploy the provider by running: `mvn package` and dropping the jar under `$KEYCLOAK_HOME/providers`

> Once started open http://localhost:8080/auth/realms/master/hello and you should see the message _Hello master_.

Reviewer: ```suggestion
When the server has started, open http://localhost:8080/auth/realms/master/hello and look for the message _Hello master_.
```
